views/partials/entries.handlebars

```diff
@@ -0,0 +1,15 @@
+<div class="columns">
+    <div class="column">
+        <div class="box">
+            <a href="#">
```

Comment:
I think the <a> tag needs a data-id attribute equal to {{id}} so the edit and delete scripts can know which entry to edit or delete.

Reply:
data-id's have been added